content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-05-01 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+# XMPP@home - 3 June 2020
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of May 2020.
+
+We started drafting every new issue in the [XSF Github repository](https://github.com/xsf/xmpp.org/pulls). 
+And we are always happy to welcome contributors. Just join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. 
+
+You have a project and (want to) write about it? Please consider sharing your news or events here, too!
+Even if you can only spend a few minutes, these would already be helpful.
+
+Tasks which need to be done on a regular basis are for example:
+
+- Aggregation of news in the XMPP universe
+- Short formulation of news and events
+- Summary of the monthly communication on extensions (XEP)
+- Review of the draft
+- Translations: especially French, German and Spanish
+
+The drafting process for each newsletter [is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+[Subscribe](https://tinyletter.com/xmpp) to receive the next edition in your inbox!
```

Comment:
We suggest you [subscribe](https://tinyletter.com/xmpp) to receive the next editions in your inbox as soon as it is published! We suggest you promote this newsletter to wohever may be interested.